frontend/src/components/GameTimer/GameTimer.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+
+import { TurnLength } from '../../model/Settings';
+import TurnTimer from './TurnTimer';
+
+interface GameTimerProps {
+  isTurnTimerVisible: boolean;
+  turnLength: TurnLength;
+}
+
+const GameTimer = ({ isTurnTimerVisible = false, turnLength }: GameTimerProps) => {
```

Comment:
Let's make this more readable - define just one parameter props and destructure it in the component's body. Also, avoid having a default value for isTurnTimerVisible (this component will be probably in just one place)

The default value is useful when many instances of component use the same value of a property, but there is one scenario when it needs to be changed.